vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -808,15 +815,35 @@ def select_experts(hidden_states: torch.Tensor,
         if use_grouped_topk:
             assert topk_group is not None
             assert num_expert_group is not None
-            topk_weights, topk_ids = grouped_topk(
-                hidden_states=hidden_states,
-                gating_output=router_logits,
-                topk=top_k,
-                renormalize=renormalize,
-                num_expert_group=num_expert_group,
-                topk_group=topk_group,
-                scoring_func=scoring_func,
-                e_score_correction_bias=e_score_correction_bias)
+            if (rocm_aiter_biased_grouped_topk
```

Comment:
Can you add a `rocm_aiter_biased_group_topk` wrapper in `rocm_aiter_fused_moe` that has the same interface as `grouped_topk`? That way we can do something like:
```
if is_rocm_aiter_moe_enabled():
    import torch.ops.vllm.rocm_aiter_biased_grouped_topk as grouped_topk
else:
   from vllm.model_executor.layers.fused_moe.fused_moe import grouped_topk
```

At the top of the file? That would remove all of the dispatching logic here AFAICT. It looks like the only dead argument you would have is `scoring_func` which I guess is assumed by the AITER kernel? If so, asserting it's "softmax" inside the wrapper would be good.